src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/loadbalancer/DiagnosableLoadBalancer.kt

```diff
@@ -0,0 +1,8 @@
+package com.atlassian.performance.tools.awsinfrastructure.api.loadbalancer
+
+import com.atlassian.performance.tools.aws.api.StorageLocation
+import com.atlassian.performance.tools.infrastructure.api.loadbalancer.LoadBalancer
+
+interface DiagnosableLoadBalancer : LoadBalancer {
```

Comment:
We could make that `LoadBalancer`-agnostic, however that would make the model more abstract and that possibly could make it harder to understand.